docs/reference/cluster_manifest.md

```diff
@@ -359,3 +359,24 @@ CPU and memory limits for the sidecar container.
 * **memory**
   memory limits for the sidecar container. Optional, overrides the
   `default_memory_limits` operator configuration parameter. Optional.
+
+## Custom TLS certificates
+
+Those parameters are grouped under the `tls` top-level key.
+
+By default, the spilo image generates its own TLS certificate during startup.
```

Comment:
I believe this part doesn't belong to the operator documentation, rather to the spilo docs. I see the point of having some rationale why one would want to use this, but since it's just reference section, maybe it's better just to get rid of it. What do you think?

Reply:
You mean, starting from "By default ...", I guess

Reply:
I merged this and improved the user documentation in 0ef214aa316eb23cf058821d916c45d64fa35047